### ISSUE TITLE
0.16 Release

### ISSUE THREAD

**Issue by davidhewitt** [PyO3/pyo3#2079]
Now that Python 3.6's end of life date was reached (on December 24th), we don't have any external release blockers for 0.16.

There's still some things I would like to see finished off before 0.16:
- [x] #1884 - let's get the remaining protocols merged and `#[pyproto]` deprecated
  - [x] Sequence Protocol - #2065 seems most promising
  - [x] Buffer Protocol - #2066 looks good enough for now
  - [x] GC Protocol - #2159
- [x] #1709 
- [ ] #2072 - a decision on whether `PyMapping` behaviour should be changed.
- [x] #2034 
- [x] #2083 

Additional nice-to-haves:
- [ ] #1979 - this is currently blocked on me, so it depends how fast I can make progress on the other bits.
- [ ] #1703 - I think new collection macros would be very convenient, so would love to see the design for these agreed upon so we can ship them.

Help with reviewing these PRs / issues so that we can merge these things for 0.16 with confidence we've got the right design would be great. I would guess that we might not have 0.16 ready until mid to late Jan given the list of things above.

**Comment by aviramha** [comment]
I think https://github.com/PyO3/pyo3/pull/2083 should be included too!

**Comment by davidhewitt** [comment]
Thanks, added!

**Comment by yodaldevoid** [comment]
Seeing as 0.16 has been released, should this issue be closed and the remaining open issues moved to a new 0.17 release tracking issue?

**Comment by davidhewitt** [comment]
Thanks for the reminder, updated milestones!
